packages/library-base/src/commonMain/kotlin/io/realm/kotlin/ext/RealmListExt.kt

```diff
@@ -42,3 +47,22 @@ public inline fun <reified T : TypedRealmObject> RealmList<T>.copyFromRealm(dept
     return this.getRealm<TypedRealm>()?.copyFromRealm(this, depth)
         ?: throw IllegalArgumentException("This RealmList is unmanaged. Only managed lists can be copied.")
 }
+
+/**
+ * Query the objects of a list by the `filter` and `arguments`.
+ *
+ * @param filter the Realm Query Language predicate to append.
+ * @param arguments Realm values for the predicate.
+ */
+// Added as an extension method as we cannot add the method `fun query(...): RealmQuery<T>` to the
```

Comment:
Nitpick... I would move this inside the method...only because it is a little annoying this comment separate the public Dokka from the public interface definition

Reply:
It absolutely doesn't fit inside the method, so moved it above the docs ... 🤷 ... is that better?